setup.py

```diff
@@ -66,8 +66,8 @@ def create_edisgo_path():
         'geoplot': ['geopandas >= 0.9.0', 'contextily', 'descartes'],
         'examples': ['jupyter'],
         'dev': ['pytest', 'sphinx_rtd_theme'],
-        'full': ['geopandas >= 0.9.0', 'contextily', 'descartes', 'jupyter', 'pytest',
-                 'sphinx_rtd_theme']
+        'full': ['geopandas >= 0.9.0', 'contextily', 'descartes', 'jupyter', 'jupyterlab',
+                 'plotly', 'pytest', 'dash', 'sphinx_rtd_theme']
```

Comment:
When running the `plot_example.ipynb` I get the following error on running `app.run_server(mode="inline",debug=True)`:
```
AttributeError: ('Read-only: can only be set in the Dash constructor or during init_app()', 'requests_pathname_prefix')
```

I tried [this suggestion](https://stackoverflow.com/a/70919276/12460232) to downgrade `dash` to `pip install dash==2.0.0` and I don't get the error anymore. However now I don't see the plots. Could you debug this?

![Screenshot at 2022-02-02 09-34-21](https://user-images.githubusercontent.com/46815982/152120056-7aa012e0-f102-401a-bf75-49a3cb3b3dba.png)

Reply:
I'm also using `dash 2.0.0`, but I see plots. Do you also use `jupyter-dash 0.4.0` ?

Reply:
yeah `jupyter-dash` is `0.4.0`. I could imagine it maybe has to do with the jupyter `build`